test/python/common.py

```diff
@@ -368,5 +394,77 @@ def _is_ci_fork_pull_request():
     return False
 
 
-SKIP_ONLINE_TESTS = os.getenv('SKIP_ONLINE_TESTS', _is_ci_fork_pull_request())
-SKIP_SLOW_TESTS = os.getenv('SKIP_SLOW_TESTS', True) not in ['false', 'False', '-1']
+def get_test_options(option_var='QISKIT_TESTS'):
+    """
+    Reads option_var from env and returns a dict in which the test options are set
+
+    Args:
+        option_var (str): The env var to read. Default: 'QISKIT_TESTS'
+
+    Returns:
+        dict: A dictionary with the format {<option>: (bool)<activated>}.
+    """
+    defaults = {
+        'skip_online': False,
+        'run_online': False,
+        'mock_online': False,
+        'skip_slow': True,
+        'run_slow': False,
+        'rec': False
+    }
+
+    def turn_true(option):
+        """
+        Turns an option to True
+        Args:
+            option (str): Turns defaults[option] to True
+
+        Returns:
+            bool: True, returns always True.
+        """
+
+        defaults[option] = True
+        return True
+
+    def turn_false(option):
+        """
+        Turns an option to False
+        Args:
+            option (str): Turns defaults[option] to False
+
+        Returns:
+            bool: True, returns always True.
+        """
+
+        defaults[option] = False
+        return True
+
+    if_true = {
+        'skip_online': (lambda: turn_false('run_online') and turn_false('rec')),
+        'run_online': (lambda: turn_false('skip_online')),
+        'mock_online': (lambda: turn_true('run_online') and turn_false('skip_online')),
+        'skip_slow': (lambda: turn_false('run_online')),
```

Comment:
Consider the test [`test_compile_run_remote`](https://github.com/1ucian0/qiskit-terra/blob/d2809ef263cb0c7283c33991660b5659de4becc8/test/python/test_compiler.py#L200). It is online but not marked as `slow_test` (this happen a lot in the codebase) so this implication is not correct.

Reply:
removed with e6a5cd8